src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/SchedulingAndDescription/index.tsx

```diff
@@ -93,12 +93,9 @@ export const SchedulingAndDescription = () => {
     (updatedVideoProperty: Partial<Video>) => {
       videoMutation.mutate(updatedVideoProperty);
     },
+    1000,
   );
 
-  useEffect(() => {
-    setDescription(video.description);
-  }, [video.description]);
```

Comment:
By removing this you are loosing the synchronization between multiple instructors connected on the dashboard at the same time.

1. Configure a live on a first browser
2. Open the same video on a different browser
3. Modify the description on the first one
4. You never see the new description on the other one